plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/holographicdisplays/HolographicDisplaysHook.java

```diff
@@ -0,0 +1,104 @@
+package net.dzikoysk.funnyguilds.feature.hooks.holographicdisplays;
+
+import eu.okaeri.commons.bukkit.holographicdisplays.Holo;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import net.dzikoysk.funnyguilds.FunnyGuilds;
+import net.dzikoysk.funnyguilds.config.PluginConfiguration;
+import net.dzikoysk.funnyguilds.config.sections.HologramConfiguration;
+import net.dzikoysk.funnyguilds.event.guild.GuildCreateEvent;
+import net.dzikoysk.funnyguilds.event.guild.GuildDeleteEvent;
+import net.dzikoysk.funnyguilds.feature.holograms.HologramsHook;
+import net.dzikoysk.funnyguilds.feature.placeholders.Placeholders;
+import net.dzikoysk.funnyguilds.guild.Guild;
+import net.dzikoysk.funnyguilds.shared.bukkit.ChatUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import panda.std.Option;
+import panda.std.Pair;
+import panda.std.stream.PandaStream;
+import panda.utilities.text.Formatter;
+
+public final class HolographicDisplaysHook extends HologramsHook implements Listener {
+
+    private final FunnyGuilds plugin;
+    private final PluginConfiguration config;
+    private final Map<Guild, Holo> holograms = new ConcurrentHashMap<>();
+
+    public HolographicDisplaysHook(String name, FunnyGuilds plugin) {
+        super(name);
+        this.plugin = plugin;
+        this.config = plugin.getPluginConfiguration();
+    }
+
+    @Override
+    public HookInitResult init() {
+
```

Comment:
Remove this blank line and everywhere else in this PR. It's awful 🤮

Reply:
No.

Reply:
Yes.